dist/images/ovnkube.sh

```diff
@@ -149,7 +149,7 @@ wait_for_event () {
 ready_to_start_node () {
 
   # See if ep is available ...
-  ovn_master_host=$(KUBECONFIG=${KUBECONFIG:-"/etc/origin/node/node.kubeconfig"} kubectl get ep -n ovn-kubernetes ovnkube-master 2>/dev/null | grep 6642 | sed 's/:/ /' | awk '/ovnkube-master/{ print $2 }')
+  ovn_master_host=$(kubectl --token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token) get ep -n ovn-kubernetes ovnkube-master 2>/dev/null | grep 6642 | sed 's/:/ /' | gawk '/ovnkube-master/{ print $2 }')
   if [[ ${ovn_master_host} == "" ]] ; then
```

Comment:
Did you test your changes? As I recall, you will need several options to be set for the above thing to work. As in

```
kubectl  -s=https://k8s_api_server_ip:k8s_api_server_port --token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token) --certificate-authority=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt get ep -n ovn-kubernetes
```